sdks/python/apache_beam/runners/direct/direct_runner.py

```diff
@@ -83,18 +83,18 @@ def accept(self, pipeline):
         pipeline.visit(self)
         return self.supported_by_fnapi_runner
 
+      def enter_composite_transform(self, applied_ptransform):
+        # The FnApiRunner does not support streaming execution.
+        if isinstance(applied_ptransform.transform,
+                      (ReadFromPubSub, WriteToPubSub)):
+          self.supported_by_fnapi_runner = False
```

Comment:
We don't need to set this for the source anymore ?

Reply:
No, this should be sufficient.